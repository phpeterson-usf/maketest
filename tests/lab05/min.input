$(repo_dir)/min_main
