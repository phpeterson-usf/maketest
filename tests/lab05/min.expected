min_c(1, 2) = 1
min_s(1, 2) = 1
min_c(100, 99) = 99
min_s(100, 99) = 99
